flink-core/src/main/java/org/apache/flink/api/common/ExecutionConfig.java

```diff
@@ -455,6 +459,33 @@ public boolean isForceAvroEnabled() {
 		return forceAvro;
 	}
 
+    /**
+     * Force Flink to use the generated serializers for POJOs.
+     */
+	public void enableCodeGeneration() {
+		forceCodeGeneration = true;
+	}
+
+	public void disableCodeGeneration() {
+		forceCodeGeneration = false;
+	}
+
+	public boolean isCodeGenerationEnabled() {
+		return forceCodeGeneration;
+	}
+
+	public void enableWrapGeneratedClasses() {
```

Comment:
I'm not sure if this is still needed, but I might be missing something. Shouldn't this always be enabled?